usaspending_api/settings.py

```diff
@@ -227,6 +227,63 @@
     },
 }
 
+# If caches added or renamed, edit clear_caches in usaspending_api/etl/helpers.py
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
+        'LOCATION': 'default-loc-mem-cache',
+    },
+    'locations': {
+        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
+        'LOCATION': 'locations-loc-mem-cache',
+    },
+}
+
+# Cache environment - 'local', 'disabled', or 'elasticache'
+CACHE_ENVIRONMENT = 'local'
+
+# Set up the appropriate elasticache for our environment
+CACHE_ENVIRONMENTS = {
+    # Elasticache settings are changed during deployment, or can be set manually
+    'elasticache': {
+        'BACKEND': 'django_elasticache.memcached.ElastiCache',
+        'LOCATION': 'ELASTICACHE-CONNECTION-STRING',
+        'TIMEOUT': 'TIMEOUT-IN-SECONDS',
+        'OPTIONS': {
+            'IGNORE_CLUSTER_ERRORS': True,
+        },
+    },
+    'local': {
```

Comment:
Oh, wait.  I see.  (?)  CACHE_ENVIRONMENTS is used to add an entry to CACHES, based on the constant CACHE_ENVIRONMENT, which is hardcoded into settings.py up in line 243, but can also (?) be overridden by the deployment script.  (And if you did want to disable the usaspending-cache for a particular run, would you just do that by temporarily changing line 243, or is there a more elegant way?)

It all could probably use clarification.